<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OperationForm</class>
 <widget class="QWidget" name="OperationForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 14pt &quot;Times New Roman&quot;;
background-color: rgb(50, 50, 50);
color: #b1b1b1;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,1,1,1,1,1,1">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(93, 93, 93);</string>
       </property>
       <property name="text">
        <string>Введите параметры операции</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(60, 60, 60);</string>
         </property>
         <property name="text">
          <string>Наименование товара</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="le_name">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(65, 65, 65);</string>
         </property>
         <property name="text">
          <string>Имя</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(60, 60, 60);</string>
         </property>
         <property name="text">
          <string>id товара</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="le_id">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(65, 65, 65);</string>
         </property>
         <property name="text">
          <string>id</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(60, 60, 60);</string>
         </property>
         <property name="text">
          <string>Сейчас на складе:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="le_current_count">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(65, 65, 65);</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(60, 60, 60);</string>
         </property>
         <property name="text">
          <string>Тип операции</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cb_operation_type">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QComboBox
{
	background-color: rgb(65, 65, 65);
}
QComboBox:hover
{
	background-color: rgb(70, 70, 70);
}
QComboBox:focus {
    
}
QComboBox::item {
    background: rgb(1, 1, 1);
}
QComboBox::item:hover {
    background: rgb(1, 1, 1);
}
QComboBox::item:selected {
    background: rgb(1, 1, 1);
}
QComboBox::drop-down 
{
    width: 0px;
    height: 0px;
    border: 0px;
}
QComboBox QAbstractItemView {
    color: rgb(250, 250, 250);    
    background-color: rgb(100, 100, 100);
    padding: 10px;
    border: 2px solid rgb(10, 10, 10);
    border-radius: 10px;
    padding-left: 5px;
    padding-right: 5px;
    selection-background: rgb(120, 120, 120);
}</string>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="currentText">
          <string>Поступление</string>
         </property>
         <property name="frame">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>Поступление</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Продажа</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Списание</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(60, 60, 60);</string>
         </property>
         <property name="text">
          <string>Дата </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="de_date">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QDateEdit{
    background-color: rgb(65, 65, 65);
	border: none
}
QDateEdit:hover{
    background-color: rgb(70, 70, 70);
}
QDateEdit::drop-down{
	background-color: rgb(75, 75, 75);
} 
QDateEdit::drop-down:hover {
	bbackground-color: rgb(80, 80, 80);
}
QDateEdit::drop-down:pressed {
	bbackground-color: rgb(120, 120, 120);
}</string>
         </property>
         <property name="wrapping">
          <bool>false</bool>
         </property>
         <property name="frame">
          <bool>false</bool>
         </property>
         <property name="accelerated">
          <bool>false</bool>
         </property>
         <property name="keyboardTracking">
          <bool>true</bool>
         </property>
         <property name="showGroupSeparator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
         <property name="date">
          <date>
           <year>2012</year>
           <month>3</month>
           <day>5</day>
          </date>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(60, 60, 60);</string>
         </property>
         <property name="text">
          <string>Кол-во</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="dsb_count">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(65, 65, 65);</string>
         </property>
         <property name="frame">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <double>99999.990000000005239</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="btn_ok">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color: rgb(75, 75, 75);
	border-radius: 1px;
}
QPushButton:hover {
     background-color: rgb(80,80, 80);
}
QPushButton:pressed {
    background-color: rgb(120,120,120);
}</string>
         </property>
         <property name="text">
          <string>Ок</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_cancel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color: rgb(75, 75, 75);
	border-radius: 1px;
}
QPushButton:hover {
     background-color: rgb(80,80, 80);
}
QPushButton:pressed {
    background-color: rgb(120,120,120);
}</string>
         </property>
         <property name="text">
          <string>Отмена</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
